sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -480,35 +481,49 @@ private bool TryGetReturnTypePropertyBindings(ITypeSymbol returnTypeSymbol, bool
                         foundHttpOutput = true;
                         bindingsList.Add(GetHttpReturnBinding(prop.Name));
                     }
-                    else if (prop.GetAttributes().Length > 0) // check if this property has any attributes
+
+                    var propAttributes = prop.GetAttributes();
```

Comment:
We shouldn't continue after this check. If the block above succeeds, we don't need keep going.